python_ta/reporters/plain_reporter.py

```diff
@@ -69,14 +71,17 @@ class PlainReporter(BaseReporter):
     _COLOURING = {}
 
     def __init__(self, source_lines=None, module_name=''):
+        """Reminder: see pylint BaseReporter for other instance variables init.
+        """
         super().__init__()
         self._error_messages = []
         self._style_messages = []
         self._source_lines = source_lines or []
         self._module_name = module_name
-        self.linter = None
         self._sorted_error_messages = defaultdict(list)
         self._sorted_style_messages = defaultdict(list)
+        self._output_file_name = 'pyta_output'
```

Comment:
The default here should be `None`. Store the default file name in a global constant.